build-utils/integration-docs-fetch-plugin.js

```diff
@@ -0,0 +1,83 @@
+/*eslint-env node*/
+/*eslint import/no-nodejs-modules:0 */
+const fs = require('fs');
+const https = require('https');
+const path = require('path');
+
+const PLATFORMS_URL = 'https://docs.sentry.io/_platforms/_index.json';
+const DOCS_INDEX_PATH = 'src/sentry/integration-docs/_platforms.json';
+
+const alphaSortFromKey = function(keyExtractor) {
+  return function(a, b) {
+    const nameA = keyExtractor(a);
+    const nameB = keyExtractor(b);
+    if (nameA < nameB) {
+      return -1;
+    }
+    if (nameA > nameB) {
+      return 1;
+    }
+
+    // names must be equal
+    return 0;
+  };
+};
+
+const transformPlatformsToList = ({platforms}) =>
+  Object.keys(platforms)
+    .map(platformId => {
+      const integrationMap = platforms[platformId];
+      const integrations = Object.keys(integrationMap)
+        .sort(alphaSortFromKey(key => integrationMap[key].name))
+        .map(integrationId => {
+          const {name, type, doc_link: link} = integrationMap[integrationId];
+          const id =
+            integrationId === '_self' ? platformId : `${platformId}-${integrationId}`;
+          return {
+            id,
+            name,
+            type,
+            link,
+          };
+        });
```

Comment:
🚲 🏠 

IMO this is slightly cleaner as:

```
.map(integrationId => {
  const {name, type, doc_link: link} = integrationMap[integrationId];
  const id =
    integrationId === '_self' ? platformId : `${platformId}-${integrationId}`;

  return { id, name, type, link, };
});
```

Reply:
🤷‍♂

Reply:
Much better! No idea why prettier left it this way. Fixed.